modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc

```diff
@@ -26,9 +21,19 @@ Only cluster administrators have access to the third-party UIs provided with {pr
 . Select *Insert Metric at Cursor* to view a list of predefined queries.
 
 . To create a custom query, add your Prometheus Query Language (PromQL) query to the *Expression* field.
++
+[NOTE]
+====
+As you type a PromQL expression, autocomplete suggestions appear in a drop-down list.
+These suggestions include functions, metrics, labels, and time tokens.
+You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression.
+You can also move your mouse pointer over a suggested item to view a brief description of that item.
+====
 
 . To add multiple queries, select *Add Query*.
 
+. To duplicate an existing query, select {kebab} next to the query, then choose *Duplicate query*.
+
 . To delete a query, select {kebab} next to the query, then choose *Delete query*.
 
 . To disable a query from being run, select {kebab} next to the query and choose *Disable query*.
```

Comment:
In the preview, I see steps 2 & 8. I know you didn't write those, so no worries if you don't want to make a change to them.  But right now, they're the only steps that start with a verb instead of an infinitive phrase, so it reads kind of awkward. For consistency, I would rephrase them to:

2 To view a list of predefined queries, select **Insert Metrics at Cursor**. 

8 To run the queries that you have created, run **Run Queries**. 


Again, I know this isn't content you originally wrote, so it's fine to leave it as is :)

Reply:
I revised the final step of the existing content for better consistency with the other steps.